scripts/utils/babelTransform.js

```diff
@@ -1,4 +1,6 @@
-const babelJest = require('babel-jest');
+const { dependRequire, dependRequireResolve } = require('./dependRequire');
+
+const babelJest = dependRequire('babel-jest');
```

Comment:
To apply against current master, this line needs to be edited to be:
```js
const babelJestMd = dependRequire('babel-jest');
const babelJest = babelJestMd.__esModule ? babelJestMd.default : babelJestMd;
```
This is so that react-app-rewired can use either babel-jest 26 or 27+, since there was a change to using esModules in babel-jest 27.